pyproject.toml

```
@@ -20,7 +20,7 @@ classifiers = [
 ]
 requires = [
     "starlette >=0.11.1,<=0.12.7",
-    "pydantic >=0.30,<=0.30.0"
+    "pydantic >=0.32.1,<=0.32.1"
```

Is *exact* pinning really required? It can create significant problems with tools such as conda which enforce version constraints.
```suggestion
    "pydantic >=0.32.1,<0.33"
```